lib/doctor/DiagnosticInfoBuilder.ts

```diff
@@ -0,0 +1,214 @@
+import { getProjectConfig } from '../projects';
+import { fetchProject } from '@hubspot/local-dev-lib/api/projects';
+import path from 'path';
+import pkg from '../../package.json';
+import { logger } from '@hubspot/local-dev-lib/logger';
+import { Environment } from '@hubspot/local-dev-lib/types/Config';
+import {
+  AccessToken,
+  AccountType,
+  AuthType,
+} from '@hubspot/local-dev-lib/types/Accounts';
+import { Project } from '@hubspot/local-dev-lib/types/Project';
+import { getAccountId } from '@hubspot/local-dev-lib/config';
+import { getAccountConfig, getConfigPath } from '@hubspot/local-dev-lib/config';
+import { getAccessToken } from '@hubspot/local-dev-lib/personalAccessKey';
+import { walk } from '@hubspot/local-dev-lib/fs';
+import util from 'util';
+import { exec as execAsync } from 'child_process';
+import process from 'process';
+
+export type ProjectConfig = Awaited<ReturnType<typeof getProjectConfig>>;
+
+const hubspotCli = '@hubspot/cli';
```

Comment:
I can't remember if I commented this last time, so ignore me if I did and you had a good reason not to. But could we import this from package.json?

Reply:
Yep, I'll update it!  I'm using the pakage.name elsewhere, so not sure how this one hung around

Reply:
Oh, now I remember why.  This value is used in a type as a key, so I need to hardcode it in at least one place, but I'll update the rest